beacon-chain/sync/context.go

```diff
@@ -69,3 +68,15 @@ func rpcContext(stream network.Stream, chain blockchain.ChainInfoFetcher) ([]byt
 		return nil, errors.New("invalid version of %s registered for topic: %s")
 	}
 }
+
+// Validates that the rpc topic matches the provided version.
+func validateVersion(version string, stream network.Stream) error {
```

Comment:
Want to add a test for this? or it's indirectly tested

Reply:
Indirectly in the test below